python/fapolicy_analyzer/ui/__main__.py

```diff
@@ -1,10 +1,36 @@
-import gi
+import logging
+logging.basicConfig(level=logging.WARNING)
+import argparse
 
+import gi
 gi.require_version("Gtk", "3.0")
 from gi.repository import Gtk
 from .main_window import MainWindow
 
+# Globals
+gbVerbose = False
 
-if __name__ == "__main__":
-    main = MainWindow()
+
+def parse_cmdline():
+    global gbVerbose
```

Comment:
I'm not seeing any reason to use a global variable.  Unless I'm missing something you only seem to be using it for the `if`  statement below which can probably be written as 
```
if args.verbose:
    logging.root.setLevel(logging.DEBUG)
    logging.debug("Verbosity enabled.")
```

Reply:
I'll pull it out, and add it if needed.

Reply:
Still and unused variable `gbVerbose` being declared

Reply:
Yep. Missed the global keyword line.

Reply:
Hopefully, that's it...

Reply:
Looks good. 👍